arrow-libs/fx/arrow-fx-coroutines/src/commonMain/kotlin/arrow/fx/coroutines/ParMap.kt

```diff
@@ -33,6 +33,23 @@ public suspend fun <A, B> Iterable<A>.parMap(
   map { async(context) { transform.invoke(this, it) } }.awaitAll()
 }
 
+public suspend fun <A, B> Iterable<A>.parMapNotNull(
+  context: CoroutineContext = EmptyCoroutineContext,
+  concurrency: Int,
+  transform: suspend CoroutineScope.(A) -> B?
+): List<B> =
+  parMap(context, concurrency) {
+    transform(it)
+  }.filterNotNull()
```

Comment:
```suggestion
  parMap(context, concurrency, transform).filterNotNull()
```

or would that break because of type inference?